VectorDist.py

```diff
@@ -0,0 +1,5 @@
+import numpy as np
+
+if __name__ == '__main__':
+    vector = np.linspace(5, 50, 10)
+    print(vector)
```

Comment:
File should end with a newline - 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline